src/client/pydaos/__init__.py

```diff
@@ -8,15 +8,12 @@
 
 import sys
 import atexit
+# pylint: disable=import-error
+from . import pydaos_shim_3 as pydaos_shim
```

Comment:
Is there anyway to rename pydaos_shim_3 to just pydaos_shim since there is no longer a python 2 shim?